sp800-63b/sec4_aal.md

```diff
@@ -34,67 +34,67 @@ AAL1 authentication SHALL occur by the use of any of the following authenticator
 
 Cryptographic authenticators used at AAL1 SHALL use approved cryptography. Software-based authenticators that operate within the context of an operating system MAY, where applicable, attempt to detect compromise (e.g., by malware) of the user endpoint in which they are running and SHOULD not complete the operation when such a compromise is detected.
 
-Communication between the claimant and verifier (using the primary channel in the case of an Out of Band authenticator) SHALL be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to MitM attacks.
+Communication between the claimant and verifier (using the primary channel in the case of an out-of-band authenticator) SHALL be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to Man in the Middle (MitM) attacks.
```

Comment:
This is the first instance of MitM in this document so probably makes sense here. But would lowercase and hyphenate it: man-in-the-middle